core/templates/dev/head/pages/collection-player-page/collection-player-page.directive.ts

```diff
@@ -250,6 +250,24 @@ oppia.directive('collectionPlayerPage', ['UrlInterpolationService',
               ctrl.collectionId);
           };
 
+          ctrl.getExplorationTitleStyle = function(index) {
```

Comment:
Instead of generating the style here, is it possible to just have this function manipulate the `left` attribute instead? The reason is that everything else is fixed, so there's not really a reason to do this.

Reply:
Done!